src/components/List.js

```diff
@@ -137,30 +245,48 @@ You cannot undo this action, and this item's purchase history will be lost.`,
                   key={result.id}
                   className={checkTime(time) ? `deactivated` : null}
                 >
-                  <span className="container">
-                    <label htmlFor={result.id}>
-                      {result.name}
-                      <input
-                        type="checkbox"
-                        disabled={checkTime(time)}
-                        defaultChecked={checkTime(time)}
-                        name={result.id}
-                        id={result.id}
-                        value={result.id}
-                        onClick={e => handleOnCheck(e, result.purchaseDates)}
-                        className="checkbox"
-                        aria-label={result.timeClass.split('-').join(' ')}
-                      />
-                      <span className={`checkmark ${result.timeClass}`}></span>
-                    </label>
-                    <button onClick={() => setDetails(result)}>Details</button>
-                    <button
-                      className="delete-btn"
-                      onClick={() => handleDelete(result)}
+                  <Paper // adds background color. If we don't want a shadow, we can set elevation to 0
+                    elevation={1}
+                    className="list-item"
+                    style={{ margin: '.2em' }}
+                  >
+                    <Grid // used to align labe/checkbox with icon buttons and to properly space
+                      container
+                      direction="row"
+                      justify="space-between"
+                      alignItems="center"
+                      className="container"
                     >
-                      x
-                    </button>
-                  </span>
+                      <Grid item>
+                        <FormControlLabel
+                          control={getCheckboxWithColor(result, time)} //sends to above function to pull back the correct color checkbox component
+                          label={result.name}
+                          disabled={checkTime(time)}
+                          id={result.id}
+                          value={result.id}
+                          aria-label={result.timeClass.split('-').join(' ')}
+                        />
+                      </Grid>
+                      <Grid item>
+                        <IconButton
+                          onClick={() => setDetails(result)}
+                          color="primary"
+                          aria-label="item details"
+                          component="span"
+                        >
+                          <MoreHorizIcon />
+                        </IconButton>
+                        <IconButton
+                          onClick={() => handleDelete(result)}
+                          color="primary"
+                          aria-label="item details"
```

Comment:
This should be something like `Delete ${result.name}`

Reply:
great catch! thank you!